run_locally.sh

```diff
@@ -0,0 +1,15 @@
+export BASE_PATH="/Users/nielsrogge/Documents/fondant_artifacts_starcoder"
```

Comment:
Can we keep those ones as relative path for now (so that they will be saved in the same folder when `run_locally.sh` is)